documentation/latest/operations/remote_data_store.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: documentation
+---
+
+# Remote data store for Flotta Edge API
+
+## Introduction
+
+By default Flotta Edge API uses Kubernetes and CRD-based storage as a source of truth about managed Edge entities -
+`EdgeDevice` and `EdgeWorkload`. All the other CRDs serve as support for better device fleet management in Kubernetes
```

Comment:
s/fleet management/management (to avoid confusion with saas)

Reply:
Done.